schnorr_fun/src/frost.rs

```diff
@@ -651,7 +732,7 @@ impl<H: Digest<OutputSize = U32> + Clone, NG> Frost<H, NG> {
     pub fn finish_keygen(
         &self,
         keygen: KeyGen,
-        my_index: usize,
+        my_index: PartyIndex,
         secret_shares: Vec<Scalar<Secret, Zero>>,
         proofs_of_possession: Vec<Signature>,
```

Comment:
Need to pass in the message that the `proofs_of_possession` should be verified under.